src/main/java/uk/ac/cam/cl/dtg/isaac/api/PagesFacade.java

```diff
@@ -419,7 +412,7 @@ public final Response getQuestion(@Context final Request request,
                                     @Context final HttpServletRequest httpServletRequest,
                                     @PathParam("question_page_id") final String questionId) {
     Map<String, List<String>> fieldsToMatch = Maps.newHashMap();
-    fieldsToMatch.put("type", Arrays.asList(QUESTION_TYPE, FAST_TRACK_QUESTION_TYPE));
+    fieldsToMatch.put("type", List.of(QUESTION_TYPE));
```

Comment:
Not your fault, but I think this should be `TYPE_FIELDNAME` instead of `"type"`.